pages/content/unit-tests/examples/full_html.md

```diff
@@ -0,0 +1,38 @@
+---
+$title: Full document example
+$order: 8
+---
+
+Start
+
+[example preview="inline" playground="true"]
```

Comment:
We should make `preview="inline" playground="true"` the default and test this case as well.

Reply:
@sebastianbenz  since the inline preview does not work for every example I rather would not enable the preview by default. There is the default_settings.md to test the default, which currently is preview=none and playground=true

Reply:
sgtm for now. We might be able to automatically determine the preview type in future.